docker-compose.development.yml

```diff
@@ -6,3 +6,6 @@ services:
     - ".:/usr/src/app"
     ports:
       - "8080:8080"
+  db:
+    ports:
+      - "3306:3306"
```

Comment:
This for my curiosity: why do we need to open the port?

Reply:
Ah it doesn't - this has been left over from a WIP branch when I was poking around the database with a GUI client.